include/exadg/fluid_structure_interaction/driver_precice.h

```diff
@@ -0,0 +1,105 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2022 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_FLUID_STRUCTURE_INTERACTION_PRECICE_DRIVER_H_
+#define INCLUDE_EXADG_FLUID_STRUCTURE_INTERACTION_PRECICE_DRIVER_H_
+
+#include <deal.II/lac/la_parallel_vector.h>
+
+// application
+#include <exadg/fluid_structure_interaction/precice/precice_adapter.h>
+#include <exadg/fluid_structure_interaction/precice/precice_parameter.h>
+#include <exadg/fluid_structure_interaction/user_interface/application_base.h>
+
+// grid
+#include <exadg/grid/get_dynamic_mapping.h>
+
+namespace ExaDG
+{
+namespace FSI
+{
+namespace preCICE
+{
+using namespace dealii;
```

Comment:
I think we should remove `using namespace dealii;` already in the present PR (according to PR #145).